jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/UserTaskService.java

```diff
@@ -494,6 +494,15 @@
      */
     Long saveContent(Long taskId, Map<String, Object> values);
     
+    /**
+     * Saves gives values as content of a task, applies to task output as input cannot be altered
```

Comment:
This method description might be reworded, as to differ saveContenFromUser from the saveContent method.